apps/backend/src/pies/repository/PieRepository.ts

```diff
@@ -0,0 +1,30 @@
+import { Pie } from '../domain/pie';
+
+export abstract class PieRepository {
+  abstract create(pie: Pie): Promise<Pie>;
+  abstract findOneByName(name: string): Promise<Pie | null>;
+  abstract findOneBySymbol(symbol: string): Promise<Pie | null>;
+  abstract findAll(): Promise<Pie[]>;
+}
+
+export class PieRepositoryStub extends PieRepository {
+  constructor(private pies: Pie[]) {
+    super();
+  }
+
+  create(pie: Pie): Promise<Pie> {
```

Comment:
just an empty line between methods here

Reply:
I added the extra lines to the rest of the methods. Note that this is something that's ignored by Prettier: it is fine with either 1 line or no lines between methods, but 2 lines are removed automatically when formatting.